test/x-describe/test.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+const assert = require('assert');
+
+xdescribe('skipped describe block', () => {
+    it('should be skipped', () => {
+        assert(false);
+    });
+
+    it('should be skipped', () => {
+        assert(false);
+    });
+});
```

Comment:
I think it's better to combine `test/x-suite` with `test/describe-skip` and rename into `suite-skip`